install-plugins.js

```diff
@@ -0,0 +1,244 @@
+    '@fastify/one-line-logger': '@fastify/one-line-logger: Formats Fastify\'s logs into a nice one-line message. See https://github.com/fastify/one-line-logger for more.',
+    '@fastify/request-context': '@fastify/request-context: Request-scoped storage, based on AsyncLocalStorage (with fallback to cls-hooked), providing functionality similar to thread-local storages. See https://github.com/fastify/fastify-request-context for more.',
+    '@fastify/response-validation': '@fastify/response-validation: A simple plugin that enables response validation for Fastify. See https://github.com/fastify/fastify-response-validation for more.',
+    '@fastify/routes': '@fastify/routes: Plugin that provides a Map of routes. See https://github.com/fastify/fastify-routes for more.',
+    '@fastify/routes-stats': '@fastify/routes-stats: Provide stats for routes using node:perf_hooks. See https://github.com/fastify/fastify-routes-stats for more.',
+    '@fastify/schedule': '@fastify/schedule: Plugin for scheduling periodic jobs, based on [toad-scheduler](https://github.com/kibertoad/toad-scheduler). See https://github.com/fastify/fastify-schedule for more.',
+    '@fastify/soap-client': '@fastify/soap-client: a SOAP client plugin for Fastify. See https://github.com/fastify/fastify-soap-client for more.',
+    '@fastify/static': '@fastify/static: Plugin for serving static files as fast as possible. See https://github.com/fastify/fastify-static for more.',
+    '@fastify/swagger': '@fastify/swagger: Plugin for serving Swagger/OpenAPI documentation for Fastify, supporting dynamic generation. See https://github.com/fastify/fastify-swagger for more.',
+    '@fastify/swagger-ui': '@fastify/swagger-ui: Plugin for serving Swagger UI. See https://github.com/fastify/fastify-swagger-ui for more.',
+    '@fastify/throttle': '@fastify/throttle: Plugin for throttling the download speed of a request. See https://github.com/fastify/fastify-throttle for more.',
+    '@fastify/under-pressure': '@fastify/under-pressure: Measure process load with automatic handling of _"Service Unavailable"_ plugin for Fastify. See https://github.com/fastify/under-pressure for more.',
+    '@fastify/url-data': '@fastify/url-data: Decorate the Request object with a method to access raw URL components. See https://github.com/fastify/fastify-url-data for more.',
+    '@fastify/view': '@fastify/view: Templates rendering (ejs, pug, handlebars, marko) plugin support for Fastify. See https://github.com/fastify/point-of-view for more.',
+    '@fastify/vite': '@fastify/vite: Integration with Vite, allows for serving SPA/MPA/SSR Vite applications. See https://github.com/fastify/fastify-vite for more.',
+    '@fastify/websocket': '@fastify/websocket: WebSocket support for Fastify. Built upon ws. See https://github.com/fastify/fastify-websocket for more.'
```

Comment:
We should be downloading this from a public resource, and not duplicate the data here. Otherwise maintaining this would be problematic.

cc @Fdawgs @Eomm, where do you think it would be better to have this metadata? Could it be part of the website?